src/redux/configureStore.js

```diff
@@ -0,0 +1,14 @@
+import { createStore, combineReducers, applyMiddleware } from 'redux';
+import logger from 'redux-logger';
+import booksReducer from './books/books';
+
+const reducer = combineReducers({
+  books: booksReducer,
+});
+
+const store = createStore(
+  reducer,
+  applyMiddleware(logger),
+);
+
+export default store;
```

Comment:
- [ ] Kindly configure your store using `configureStore` inside `redux toolkit` instead of `createStore` to achieve this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/add_reducers_an_actions.md#:~:text=Configure%20the%20Redux,using%20configureStore%20function.).

Reply:
thank you so mush I will update it soon.